utils.js

```diff
@@ -74,7 +74,12 @@ const MENTOR_STATUSES = {
   pending: 'pending',
 };
 
+const ALLOWED_MENTOR_STATUSES = {
+  active: 'active',
```

Comment:
Let's reuse `MENTOR_STATUSES` statuses enum instead of hard-coding the statuses once again